superset/connectors/sqla/models.py

```diff
@@ -461,7 +460,7 @@ def make_sqla_column_compatible(
         if db_engine_spec.allows_column_aliases:
             label = db_engine_spec.make_label_compatible(label_expected)
             sqla_col = sqla_col.label(label)
-        sqla_col._df_label_expected = label_expected
+        sqla_col._df_label_expected = label_expected  # pylint: disable=protected-access
```

Comment:
Should we make it public or accessible through a property?

Reply:
Unfortunately the type on this is `Column`, which comes direct from SQLAlchemy. I'd prefer not to monkeypatch their internals.